documentation/b2b.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: master
+permalink: /documentation/b2b/
+title: B2B
+weight: 7
+---
+# B2B Storefront Configuration and Implementation
+
+## Configuration
+TBA
+
+## KeyCloak Theme Implementation
+By default, KeyCloak provides a set of themes that may be used for the various actions KeyCloak supports (login, accounts, etc). Either of these existing themes may be used for the supported flows, or a customized theme may be added to the KeyCloak Docker Image if desired. The REACT PWA Reference Storefront includes a login theme that may be used for this purpose.
```

Comment:
**About this task**
By default, KeyCloak provides a set of themes, such as, login or accounts, that can be used for the various actions that KeyCloak supports.
Use one of these existing themes for the supported flows or add a customized theme to the KeyCloak Docker Image, if required. The REACT PWA Reference Storefront provides a login theme to use for (what?)

Reply:
The actual theme name is "login theme". Others are "account theme", etc